example-react-app/src/app/associationM2M-management/AssociationM2MEditor.tsx

```diff
@@ -96,13 +97,7 @@ const AssociationM2MEditor = observer(
           form={form}
           validateMessages={createAntdFormValidationMessages(intl)}
         >
-          <Field
-            entityName={ENTITY_NAME}
-            propertyName="name"
-            formItemProps={{
-              style: { marginBottom: "12px" }
-            }}
-          />
+          <Fields relationOptions={relationOptions} entityId={entityId} />
```

Comment:
I think the generated code should remain as it is.

What we need to do is to extract a portion of a template and then reuse it across templates. I think it can be done by using EJS [includes](https://github.com/tj/ejs#includes).